src/sinol_make/commands/init/__init__.py

```diff
@@ -23,21 +27,35 @@ def configure_subparser(self, subparser: argparse.ArgumentParser):
             description='Create package from predefined template with given id.'
         )
         parser.add_argument('task_id', type=str, help='id of the task to create')
-        parser.add_argument('directory', type=str, nargs='?',
+        parser.add_argument('-o', '--output', type=str,
             help='destination directory to copy the template into, defaults to task_id')
         parser.add_argument('-f', '--force', action='store_true',
             help='overwrite files in destination directory if they already exist')
+        parser.add_argument('-t', '--template', nargs='+', default=[self.DEFAULT_TEMPLATE, self.DEFAULT_SUBDIR],
+            help='specify template repository or directory, optionally subdirectory after space'
+                f' (default: {self.DEFAULT_TEMPLATE} {self.DEFAULT_SUBDIR})')
+        parser.add_argument('-v', '--verbose', action='store_true')
         return parser
 
-    def download_template(self):
-        repo = 'https://github.com/sio2project/sinol-make.git'
-        package_dir = 'sinol-make/example_package'
+    def download_template(self, template_paths = [DEFAULT_TEMPLATE, DEFAULT_SUBDIR]):
+        template = template_paths[0]
+        subdir = template_paths[1] if len(template_paths) > 1 else ''
+
         self.used_tmpdir = tempfile.TemporaryDirectory()
         tmp_dir = self.used_tmpdir.name
-        ret = subprocess.run(['git', 'clone', '-q', '--depth', '1', repo], cwd=tmp_dir)
-        if ret.returncode != 0:
-            util.exit_with_error("Could not access repository. Please try again.")
-        path = os.path.join(tmp_dir, package_dir)
+
+        is_url = template.startswith("https://") or template.startswith("git@")
```

Comment:
What about `ssh://`?

Reply:
I didn't realize other schemes existed 🤔 
Added some more to the list